lib/security/NtlmSecurity.js

```diff
@@ -0,0 +1,32 @@
+"use strict";
+
+var _ = require('lodash');
+
+function NtlmSecurity(username, password, domain, workstation) {
```

Comment:
Same here, please stay close to the already established naming convention and rename to `NTLMSecurity`